Framework/LiveData/inc/MantidLiveData/ADARA/ADARAParser.h

```diff
@@ -29,6 +29,29 @@ class DLLExport Parser {
   /// Destructor
   virtual ~Parser();
 
+  long last_bytes_read;
```

Comment:
Yeah, that's probably better.  I take it we actually have int64_t (and the others) on Windows?